README.md

```
@@ -0,0 +1,34 @@
+2. **TELEGRAM_BOT_TOKEN** ve **CHAT_ID**: Telegram botu üzerinden bildirim gönderebilmek için gerekli bilgiler.
+
+### Kullanım:
+1. Python ortamınızı hazırlayın ve gerekli kütüphaneleri yükleyin:
+   ```bash
+   pip install requests beautifulsoup4
+   ```
+2. Botunuzu ve Telegram kanalınızı ayarlayın.
+3. Kullanıcı adı, şifre, bot token ve chat id bilgilerini doğru şekilde girin.
+4. Betiği çalıştırarak, vize randevularını izlemeye başlayın.
+
+### Uyarılar:
```

Markdownlint (0.37.0)

Punctuation: ':'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)